deny.toml

```diff
@@ -13,7 +13,8 @@ unlicensed = "deny"
 allow = [
     "MIT",
     "Apache-2.0",
-    "0BSD"
+    "0BSD",
+    "Unicode-DFS-2016"
```

Comment:
not sure why this didn't get flagged before

Reply:
ah, need to invoke `cargo deny --all-features check`, fixed in 3bfb0b51ebf9f3f663dbf5d132187d67ca4bacbf